app/views/content/trainees/start-your-teacher-training/_content.html.erb

```diff
@@ -0,0 +1,53 @@
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Developing pupils</h2>
+    <div class="inset">
+      <p>During training, you'll get to find out what it takes to become the kind of teacher that pupils will remember years after leaving school.</p>
+      <p>Your lessons will be unobserved and observed, giving you plenty of chances to see pupils experience 'aha' moments, when your teaching helps them learn something new.</p>
+      <p>You'll be given the tools needed to inspire, motivate and challenge pupils. You'll find out how to adapt your teaching style to suit the strengths and needs of all pupils, and how to manage behaviour when things go off script.</p>
+      <p>Ultimately, you'll get to see how your teaching can help pupils progress and get the outcomes you'll know they are capable of.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Developing you</h2>
+    <div class="inset">
+      <p>You'll learn from teachers like you through taught sessions, seminars and workshops. You'll have the chance to get involved in academic or professional research so you can dig deeper into what makes you tick as a teacher.</p>
+      <p>You'll be able to see how you're getting on by completing tasks and assignments, so you can track what you're learning as you're teaching. These will be a combination of less formal, ongoing assessments, and formal assessments that usually happen at the end of your course.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Working with a mentor</h2>
+    <div class="inset">
+      <p>You won't be taking on this amazing challenge alone. You'll team up with a mentor to support you in your school placements. Your mentor will guide you through your training, sharing learning and career development opportunities with you.</p>
+      <p>Your mentor will meet with you regularly to make sure your training is a positive experience.</p>
+    </div>
+  </section>
+ </div>
+
+<div class="row">
+  <div class="grey">
+    <section class="col col-720">
+      <h2 class="heading--box-pink">Any questions? Speak to your training provider</h2>
+      <div class="inset">
+        <p>Your provider will be able to answer any questions you have about your training.</p>
+        <p>Providers can answer practical questions about your placement, including:</p>
+        <p><ul>
+        <li>what a typical training term is like</li>
+        <li>how much time is spent in school or on academic learning</li>
+        <li>how you'll be assessed with exams and coursework</li></ul>
+        </p>
+        <p>Feeling prepared is half the challenge, so if you have anything you feel unsure about, speak to your provider.</p>
+      </div>
```

Comment:
@Sarah-DfE Should we end with more of a positive note - eg `Good luck with your training!` as per email or similar?